### PR TITLE
Remove Dynamic Types

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Comment by rsanchez15** [comment]
This comment should be updated with the latest Fast DDS Discovery Server test results.

Test results:

- [discovery_server_linux/2993](http://jenkins.eprosima.com:8080/job/discovery_server_linux/2993/)

File: test/configuration/test_cases/test_18_disposals_remote_servers_multiprocess.xml [hunk 1]
```diff
@@ -67,17 +67,8 @@
...
-            <type>
-              <struct name="sample_type_1">
-                <member name="index" type="uint32" />
-                <member name="message" type="string" />
-              </struct>
-            </type>
-        </types>
-
      </profiles>
 </DS>
```
Comment (jparisu) [inline test/configuration/test_cases/test_18_disposals_remote_servers_multiprocess.xml:74]: *In general*
Add EOF in all files that need it

**Review by jparisu — APPROVED** [review]
LGMT
(some EOF are missing, but it is not important)

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**